website/content/docs/platform/k8s/csi/index.mdx

```diff
@@ -5,6 +5,8 @@ description: >-
   The Vault CSI Provider allows pods to consume Vault secrets using CSI volumes.
 ---
 
+-> **Deprecated:** The issuer parameter has been deprecated as of Vault 1.9 and will and will be removed in a future release.
+
```

Comment:
Should this be at the top of the page? I'm thinking it would be good at the top of the "Setting `issuer` for Kubernetes authentication" section.